hedyorg/hedy
# 🖊️ Add support for calculations with play command from level 6 #5044

Fixes #5044

**How to test**
- Check if the added unit tests cover correctly the added logic
- Start hedy locally and navigate to level 6. Execute the following command and ensure the three tones are played:
```
play_number = 34
play play_number
play play_number + 1
play play_number + 2
```
